client/audiodevs/PipeWire/pipewire.c

```diff
@@ -368,7 +368,11 @@ static size_t pipewire_playbackLatency(void)
   struct pw_time time = { 0 };
 
   pw_thread_loop_lock(pw.thread);
+#if PW_MAJOR >= 0 && PW_MINOR >= 3 && PW_MICRO >= 50
```

Comment:
This logic is wrong. For example, PipeWire 1.0.0 will fail this check and fall back to the legacy code.

Reply:
You can use the `PW_CHECK_VERSION` macro for this btw.

Reply:
You're right. fixed.